src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3368,12 +3368,52 @@ public async Task AbortMigration_Returns_True_On_Success()
             .ReturnsAsync(response);
 
         // Act
-        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationSourceId);
+        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationId);
 
         // Assert
         expectedBooleanResponse.Should().Be(actualBooleanResponse);
     }
 
+    [Fact]
+    public async Task AbortMigration_Surfaces_Error_With_Incorrect_Migration_ID()
+    {
+        // Arrange
+        const string migrationId = "1234";
+
+        const string query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        const string gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    })
+                   { success }";
+
+        var payload = new
+        {
+            query = $"{query} {{ {gql} }}",
+            variables = new
+            {
+                migrationId,
+            },
+            operationName = "abortRepositoryMigration"
+        };
+
+        const string actualErrorResponse = $"Invalid migration id: {migrationId}";
+
+        // Act
+        await _githubApi.AbortMigration(migrationId);
```

Comment:
This won't work here, whenever a test is testing for an exception being thrown the correct way to test is the way you did below so this like can be safely deleted it not it will throw the actual exception and the test will fail.

Reply:
There are [several](https://github.com/github/gh-gei/blob/142c9f8246aef888488283d11d4bc50fe938f85a/src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs#L88) of other examples in the Tests file that do this, why does it not work in this particular scenario?